Riot to element update

Shrinks99 opened the pull request:
Updates Jekyll gems to latest version, changes language on the Get Involved page to reflect Element's new name and login flows.  Closes #269 

Travis is just broken because of broken Freifunk HTTPS cert.

Shrinks99 committed:
Updates Jekyll Gems to current versions

Shrinks99 committed:
Updates to reflect Element's new name and login flow

darkdrgn2k reviewed (approved):
Still say we should add irc and slack like we did in contacts

benhylau commented:
Can we revise our `Get Involved` a little to clarify Matrix and bias towards leading people to our active Toronto Community Network project?

- We recommend using the [Element mobile or desktop app](https://element.io):
  - If you already have a Matrix account, simply join `#tomesh:tomesh.net` and introduce yourself
  - If you do not have a Matrix account, select _Create Account_, select _Other_ and enter `https://matrix.tomesh.net` as the homeserver URL to create a new account on our Toronto Mesh homeserver
- Alternatively, you can connect through one of our bridged services (but this is not recommended as not all our rooms are bridged)

Change this line to lead people to our TCN README and direct them to email us and intro themselves in the email. This should be the first thing on that page.

>1. Review the current list of [projects]({{ site.baseurl }}/projects) and [work with us](#work-with-us)
>    - Visit our [GitHub](https://github.com/tomeshnet) used to host our open-source software and all important documents

Shrinks99 committed:
Updates Slack information

- Clarifies the downsides of Slack vs Element
- Updates the Slack icon in icon.slack.html
- Updates icon-slack.html to link to our Slack bridge instead of CivicTech's
- Removes the links to Matrix communities because they don't work as well as they should and haven't for years, replaced with the link to our general chat room

Shrinks99 committed:
Updates concurrent-ruby

The last gem update will fix our security vulnerability when this is merge.

Tested and working as of August 8th

Shrinks99 committed:
Removed "Contribute to our Git Repos" section

Lots of this information is duplicated across our website and also not nessisairly relevant to our new onboarding process.

- Added TCN readme as suggested by @benhylau
- I don't think we've ever had a donation jar ever
- added line about grants because we're actually getting one!

Shrinks99 commented:
Okay this should be good to go and should also fix the security issues with the gem updates!  Travis is upset because http://tao.ca and http://radicalnetworks.org don't exist anymore :(

Shrinks99 committed:
Merge branch 'master' into riot-to-element-update
